django_usaid_clinicfinder/requirements.txt

```diff
@@ -1,18 +1,29 @@
+amqp==1.4.9
+anyjson==0.3.3
+billiard==3.3.0.23
+celery==3.1.23
+contextlib2==0.5.1
+cookies==2.2.1
+dj-database-url==0.4.0
 Django==1.7.2
```

Comment:
Perhaps use Django==1.7.11 instead? It is still an unsupported version but at least it includes the next 9 security releases.